model/riscv_fdext_regs.sail

```diff
@@ -193,11 +193,8 @@ function wF (r : regno, in_v : flenbits) -> unit = {
   };
 
   dirty_fd_context();
-
-  if   get_config_print_reg()
-  then
-      /* TODO: will only print bits; should we print in floating point format? */
-      print_reg("f" ^ dec_str(r) ^ " <- " ^ FRegStr(v));
+  
+  freg_write_callback(regno_to_regidx(r), in_v);
```

Comment:
The default callback needs to print the register otherwise this removes tracing capabilities from the model.

Reply:
Added an implementation of the callbacks for trace printing to the `riscv_types.sail`, `riscv_csr_begin.sail`, `riscv_vreg_type.sail` and call them from the default callbacks in the `riscv_default_callbacks.c`.